--- origsrc/evince-2.22.2/configure.ac	2008-05-28 16:14:35.000000000 -0500
+++ src/evince-2.22.2/configure.ac	2008-06-01 00:19:41.671875000 -0500
@@ -60,7 +60,7 @@
 PKG_CHECK_MODULES(FRONTEND_CORE, gtk+-2.0 >= $GTK_REQUIRED libglade-2.0 gthread-2.0 gio-2.0 >= $GLIB_REQUIRED)
 PKG_CHECK_MODULES(SHELL_CORE, libxml-2.0 >= $LIBXML_REQUIRED gtk+-2.0 >= $GTK_REQUIRED gtk+-unix-print-2.0 >= $GTK_REQUIRED gio-2.0 >= $GLIB_REQUIRED gnome-icon-theme >= $GNOME_ICON_THEME_REQUIRED gthread-2.0 libglade-2.0 gconf-2.0)
 
-BACKEND_LIBTOOL_FLAGS="-module -avoid-version"
+BACKEND_LIBTOOL_FLAGS="-module -avoid-version -no-undefined"
 AC_SUBST(BACKEND_LIBTOOL_FLAGS)
 
 dnl ===== Check special functions
--- origsrc/evince-2.22.2/libdocument/Makefile.am	2008-05-28 16:14:02.000000000 -0500
+++ src/evince-2.22.2/libdocument/Makefile.am	2008-06-01 00:12:36.328125000 -0500
@@ -8,7 +8,7 @@
 
 lib_LTLIBRARIES = libevbackend.la
 
-libevbackend_la_LDFLAGS = -export-dynamic
+libevbackend_la_LDFLAGS = -export-dynamic -no-undefined
 libevbackend_la_LIBADD = $(LIB_LIBS)
 
 NOINST_H_FILES =				\
