--- origsrc/evince-2.26.2/libdocument/Makefile.am	2009-09-08 21:22:17.021246200 -0500
+++ src/evince-2.26.2/libdocument/Makefile.am	2009-09-08 20:14:01.027158300 -0500
@@ -134,7 +134,7 @@ EvinceDocument-$(EV_API_VERSION).gir: li
 	--nsversion=$(EV_API_VERSION) \
 	--include=Gio-2.0 \
 	--include=Gtk-2.0 \
-	--library=evdocument \
+	--library=libevdocument.la \
 	--libtool="$(LIBTOOL)" \
 	--output $@ \
 	--pkg gobject-2.0 \
@@ -144,7 +144,7 @@ EvinceDocument-$(EV_API_VERSION).gir: li
 	-I$(top_srcdir) \
 	-I$(top_builddir) \
 	-DEVINCE_COMPILATION \
-	$(INST_H_FILES)
+	$(addprefix $(srcdir)/,$(INST_H_FILES))
 
 girdir = $(GIRDIR)
 gir_DATA = EvinceDocument-$(EV_API_VERSION).gir
--- origsrc/evince-2.26.2/libview/Makefile.am	2009-09-08 21:22:17.024996300 -0500
+++ src/evince-2.26.2/libview/Makefile.am	2009-09-08 20:49:16.923740700 -0500
@@ -107,8 +107,8 @@ EvinceView-$(EV_API_VERSION).gir: libevv
 	--nsversion=$(EV_API_VERSION) \
 	--include=Gio-2.0 \
 	--include=Gtk-2.0 \
-	--include=EvDocument-$(EV_API_VERSION) \
-	--library=evview \
+	--include=EvinceDocument-$(EV_API_VERSION) \
+	--library=libevview.la \
 	--libtool="$(LIBTOOL)" \
 	--output $@ \
 	--pkg gobject-2.0 \
@@ -118,7 +118,7 @@ EvinceView-$(EV_API_VERSION).gir: libevv
 	-I$(top_srcdir) \
 	-I$(top_builddir) \
 	-DEVINCE_COMPILATION \
-	$(INST_H_FILES)
+	$(addprefix $(srcdir)/,$(INST_H_FILES))
 
 girdir = $(GIRDIR)
 gir_DATA = EvinceView-$(EV_API_VERSION).gir
